plugins/reolink/src/main.ts

```diff
@@ -84,20 +86,15 @@ class ReolinkCamera extends RtspSmartCamera implements Camera, Reboot, Intercom,
     }
 
     async getObjectTypes(): Promise<ObjectDetectionTypes> {
-        if (this.storageSettings.values.doorbell && this.storageSettings.values.doorbellUseOnvifDetections) {
-            return {
-                classes: ['person'],
-            };
-        }
-
         try {
-            const ai: AIState = this.storageSettings.values.hasObjectDetector?.value;
+            const client = this.getClient();
+            const ai = await client.getAiState();
```

Comment:
getObjectTyeps should complete quickly and generally never fail. The intent of the previous was to cache the results. This change needs to be rolled back as this will cause UI hangs if the doorbell is offline.

Reply:
Ahh, I see.  Changed.  Thanks for the feedback!